web/src/containers/ActivityDetailContractorResources.js

```diff
@@ -22,7 +23,15 @@ Label.propTypes = {
   children: PropTypes.node.isRequired
 };
 
+const DOC_TYPES = ['Contract', 'RFP'];
+
 class ContractorExpenses extends Component {
+  state = { docType: DOC_TYPES[0] };
```

Comment:
What's this?  I've always seen state initialized in the constructor using `this.state`. Is this a way of initializing a class member variable?

Reply:
just another way to initialize state without needing to use constructor :)